src/org/rascalmpl/library/lang/rascal/syntax/Rascal.rsc

```diff
@@ -210,7 +210,7 @@ syntax Expression
 	| \list           : "[" {Expression ","}* elements0 "]"
 	| reifyType      : "#" Type type !>> "[" !selector
 	| range          : "[" Expression first ".." Expression last "]"
-	| \tuple          : "\<" {Expression ","}+ elements "\>" 
+	| \tuple          : "\<" {Expression ","}* elements0 "\>" 
```

Comment:
Just to make sure: did you run a check to make sure it didn't introduce new ambiguities? (for example, run it against the rascal files of rascal/typepal/rascal-core?)

Reply:
Yes according to Dr ambiguity this doesn't introduce new ambiguity. That is empirical evidence based on random input generation. However I can't guarantee there isn't any ambiguity and in that case we might find out when we test this on Rascal-core. That would be surprising since that code would have to contain `<>` somewhere already.